wire-ios/Wire-iOS Tests/DeviceManagement/MockEnrolE2eICertificateUseCase.swift

```diff
@@ -20,9 +20,9 @@ import Foundation
 
 final class MockEnrolE2eICertificateUseCase: EnrollE2EICertificateUseCaseProtocol {
```

Comment:
suggestion: there's a mock in request strategy for it, is it possible to use it in including WireRequestStrategySupport in the tests target?